sentinel/src/transitions.js

```diff
@@ -54,6 +57,14 @@ const changeQueue = async.queue((change, callback) => {
 
     updateMetadata(change, callback);
   });
+
+});
+
+changeQueue.drain(() => {
```

Comment:
I'm presuming drain's callback gets called multiple times, and only after it's been filled once, but this is a guess